s2clientprotocol/sc2api.proto

```diff
@@ -63,6 +63,8 @@ import "s2clientprotocol/ui.proto";
 //                  | ended                                             |                       |
 // save_replay      | in_game                                           |                       |
 //                  | ended (only after a game)                         |                       |
+// map_command      | in_game                                           |                       |
+//                  | ended                                             |                       |
```

Comment:
Are you sure this is possible in the ended state? I thought you need to call restart first.